packages/design-system/src/components/RichRadioButton/RichRadioButton.component.tsx

```diff
@@ -0,0 +1,69 @@
+// eslint-disable-next-line @talend/import-depth
+
+import { RichRadioButtonProps } from './RichRadioButton.types';
+import { StackVertical, StackHorizontal } from '../Stack';
+import { getTagVariant } from './RichRadioButton.utils';
+import { getIconWithDeprecatedSupport } from '../Icon/DeprecatedIconHelper';
+import style from './RichRadioButton.module.scss';
+
+const RichRadioButton = ({
+	dataFeature,
+	description,
+	asset,
+	id,
+	isChecked = false,
+	isDisabled = false,
+	isReadOnly = false,
+	name,
+	onChange,
+	tags,
+	title,
+}: RichRadioButtonProps) => {
```

Comment:
could be _cool_ if the component could manage its loading/refreshing state, wdyt ?

Reply:
Is there any loading status visible in the figma specs ?

Reply:
There is not and I've discussed it with Maxime. We won't do it for now as we don't have a way to know what content the card will have.